examples/spring-streams-example/src/test/kotlin/com/stove/spring/streams/example/e2e/exampleTest.kt

```diff
@@ -0,0 +1,109 @@
+package com.stove.spring.streams.example.e2e
+
+import arrow.core.Option
+import com.google.protobuf.Message
+import com.trendyol.stove.testing.e2e.standalone.kafka.kafka
+import com.trendyol.stove.testing.e2e.system.TestSystem
+import io.confluent.kafka.streams.serdes.protobuf.KafkaProtobufSerde
+import kotlinx.coroutines.runBlocking
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import stove.example.protobuf.input1
+import stove.example.protobuf.input2
+import stove.example.protobuf.output
+import stove.example.protobuf.Input1Value.Input1
+import stove.example.protobuf.Input2Value.Input2
+import stove.example.protobuf.OutputValue.Output
+import java.util.UUID
+import kotlin.time.Duration.Companion.seconds
+
+@ExtendWith(Init::class)
+class ExampleTest : StoveBase() {
```

Comment:
Instead of inheriting StoveBase, you can just use `FunSpec` from Kotest and write the code with kotest. Entire project uses Kotest as the design decision. We need a very strong reason to use Junit's APIs. In this case, I believe you can just use Kotest framework for testing.

Reply:
done